tests/i2c_slave_checker.py

```diff
@@ -14,8 +14,8 @@ def __init__(self, scl_port, sda_port, speed,
         self._sda_port = sda_port
         self._tsequence = tsequence
         self._speed = speed
-        self._bit_time = 1000000 / speed
-        print "Checking I2C: SCL=%s, SDA=%s" % (self._scl_port, self._sda_port)
+        self._bit_time = 1000000e6 / speed
```

Comment:
is there not a constant on px for this value? If not that sounds like a hole!

Reply:
I didn't find any. Its hardcoded in pyxsim as well, https://github.com/xmos/test_support/blob/develop/lib/python/Pyxsim/pyxsim.py#L257